ar/src/semantic/bundle.cpp

```diff
@@ -41,13 +41,15 @@
  *
  ******************************************************************************/
 
+#include <iostream>
 #include <ikos/ar/semantic/bundle.hpp>
 #include <ikos/ar/semantic/code.hpp>
 #include <ikos/ar/semantic/context.hpp>
 #include <ikos/ar/semantic/data_layout.hpp>
 #include <ikos/ar/semantic/function.hpp>
 #include <ikos/ar/semantic/value.hpp>
 #include <ikos/ar/support/assert.hpp>
+#include <ikos/ar/format/text.hpp>
```

Comment:
I don't think we need the `#include`s you added?

Reply:
You are right. Deleted. Note these current changes were part of larger changed which included adding a "dmp()" method that was equivalent to dump(std::cout). I decided against adding dmp() methods as the equivalent could be achieved with a gdb macro.